Lesson5/Task3/src/Person.java

```diff
@@ -0,0 +1,64 @@
+import Jackets.Jacket;
+import Shoes.Shoes;
+import Trousers.Trousers;
+
+public class Person implements iPerson {
```

Comment:
С большой буквы название интерфейсов.
```suggestion
public class Person implements IPerson {
```